src/main/java/org/troyargonauts/Robot.java

```diff
@@ -18,26 +26,57 @@
  * the package after creating this project, you must also update the build.gradle file in the
  * project.
  */
+
+/**
+ *
+ */
 public class Robot extends TimedRobot {
     private Command autonomousCommand;
     
     private RobotContainer robotContainer;
 
     static PneumaticsSystem pneumatics;
 
+    private static Limelight limelight;
+
+    private Pigeon2 pigeon;
+    private WPI_Pigeon2 pigeon_wpi;
+
+
+
     @Override
     public void robotInit() {
         // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
         // autonomous chooser on the dashboard.
         robotContainer = new RobotContainer();
 
         pneumatics = new PneumaticsSystem();
+
+        limelight = Limelight.getInstance();
+
+        limelight.setLedMode(Limelight.LightMode.ON);
+        limelight.setCameraMode(Limelight.CameraMode.VISION);
+
+        pigeon= new Pigeon2(11);
+
+
+        pigeon.configFactoryDefault();
+        pigeon.clearStickyFaults();
+        final Pigeon2Configuration pigeonConfig = new Pigeon2Configuration();
+        pigeonConfig.MountPosePitch = 0;
+        pigeonConfig.MountPoseRoll = 0;
+        pigeonConfig.MountPoseYaw = 0;
+        pigeon.configAllSettings(pigeonConfig);
     }
 
     @Override
     public void robotPeriodic()
     {
         CommandScheduler.getInstance().run();
+        SmartDashboard.putNumber("Limelight Distance", limelight.getTy());
```

Comment:
Make a periodic() method with an override and put the numbers to SmartDashboard there.